ixa-fips/src/states.rs

```diff
@@ -0,0 +1,276 @@
+/*!
+
+An enum for states as represented by FIPS Geographic Region Codes. Note that the `FIPSCode` encoded type only uses six
+bits to encode the state code, which can accommodate codes <= 63. Thus, it is best to only use `FIPSCode` for actual
+proper states.
+
+*/
+
+use crate::StateCode;
+use std::fmt::Display;
+
+#[derive(Copy, Clone, Eq, PartialEq, Hash, Debug)]
+pub enum USState {
+    AL = 1,
+    AK = 2,
+    AmericanSamoa = 3, // (FIPS 5-1 reserved code)
```

Comment:
FIPS 5-1 seems to be an outdated version of the spec, these do have two-letter abbreviations you could use